lib/common/utils.js

```diff
@@ -269,15 +269,76 @@ exports.createRoutes = function createRoutes(files, srcDir) {
   return cleanChildrenRoutes(routes)
 }
 
+/**
+ * Removes a module from the cache.
+ * based on https://github.com/dwyl/decache
```

Comment:
If this module is working, can't we use `decache` npm module itself?

Reply:
Yes, should be as easy as:
- add "decache" to dependencies 
- use "decache()" instead of "rmCache" in builder.js 

The reason why I didn't do it was:
1.) didn't want to introduce a new depencdy
2.) didn't like the way how the build the module - "overwrite"/extending require (but just my personal opinion ;))

If you want me to implement it, just let me know.